Documentation/learning/glossary.md

```diff
@@ -26,10 +30,93 @@ Peer is another member of the same cluster.
 
 A proposal is a request (for example a write request, a configuration change request) that needs to go through raft protocol.
 
+## Quorum
+
+The number of members of a group/cluster required to be present to perform a set of instructions/operations, usually a majority.
```

Comment:
@xiang90 i had a doubt, for example when members are 6 the quorum should be 4 based on N/2+1. Do we have a special handling for this to keep it 3 only?